### PR TITLE
Remove duplicate imports

### PR DESCRIPTION
**Pull request by liketic**
Hi, this is a mirror PR which removed some duplicate imports. And also I suggest we remove all unused imports but it's a big change. Happy to hear your comments. Thanks.

### CODE REVIEW

**Comment by jasontedor** [comment]
> And also I suggest we remove all unused imports but it's a big change.

We have some active feature branches. This might introduce a lot of merge conflicts. Let's not. 

We appreciate these cleanup PRs but we have a lot of adopt me/low-hanging fruit issues which would be great to pick up if you were interested. Let us know, we'd be happy to guide you through one.

**Comment by likel** [comment]
@jasontedor That's great! Please get me involved. I'm happy to contribute.Thanks.

File: plugins/discovery-ec2/src/main/java/org/elasticsearch/discovery/ec2/AwsEc2UnicastHostsProvider.java [hunk 1]
```diff
@@ -19,12 +19,6 @@
 
 package org.elasticsearch.discovery.ec2;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
```
Comment (javanna) [inline plugins/discovery-ec2/src/main/java/org/elasticsearch/discovery/ec2/AwsEc2UnicastHostsProvider.java]: not sure what happened but these are now needed in this class

**Comment by javanna** [comment]
this seems to be outdated now, I manually pushed the ConfigurationUtils import removal to master and 5.x

**Pull request closed without merging**